templates/template/scripts/package-ci

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
A new script I wrote to test changes as part of this rewrite. This isn't plugged into CI,  but it's a nice-to-have in case we run into issues.

Reply:
Is this intended to run to verify that user manually run through all the steps before they are submitting PRs?

Reply:
Yea whats point of this exactly.

Reply:
This isn't really intended to be run by users. It's moreso intended to be run as a sanity check by someone who wants to make a breaking change to the charts-build-scripts and wants to assess whether the repository that is absorbing the changes can be safely migrated (e.g. all commands, except validate, work).

Mostly just built it because I needed the ability to test this, but then checked it in since it seemed like it would be useful in the future.